cli/util/uploader.go

```diff
@@ -394,7 +400,8 @@ func updateIndexFiles(ctx *cli.Context, p *pool.Pool, containerID cid.ID, accoun
 		}
 		attrs := []object.Attribute{
 			*object.NewAttribute(attributeKey, strconv.Itoa(int(i))),
-			*object.NewAttribute("size", strconv.Itoa(int(indexFileSize))),
+			*object.NewAttribute("Size", strconv.Itoa(int(indexFileSize))),
```

Comment:
Better some `IndexSize` or `IndexLength`, `Size` attribute is too easy to confuse with object size.